### PR TITLE
bears: Add swift bear

### PR DESCRIPTION
**Pull request by sudheesh001**

### CODE REVIEW

**Comment by sils1297** [comment]
@sudheesh001 are you still working on this?

**Comment by sudheesh001** [comment]
Yup, I am but strangely as I mentioned the bear doesn't seem to be running as expected, I'll rebase and get back to this today.

File: tests/swift/SwiftLintBearTest.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+bad_file = """func swiftLintTest() {
+\tlet someForceCast = NSNumber() as !Int
+\tlet colonOnWrongSide : Int = 0
+\t// Smart enough to ignore comments in the code
+\t"let colonOnWrongSide : Int = 0" //
+}""".splitlines(True)
+
+SwiftLintBearTest = verify_local_bear(SwiftLintBear,
+                                      valid_files=(good_file,),
+                                      invalid_files=(bad_file,))
```
Comment (Adrianzatreanu) [inline tests/swift/SwiftLintBearTest.py]: hey i think you should add a newline at the end of the file

File: bears/swift/SwiftLintBear.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "warning": RESULT_SEVERITY.NORMAL
+    }
+
+    def run(self, filename, file):
+        '''
+        Checks the code with swiftlint. This will run swiftlint over
+        each file seperately.
+        '''
+        self.arguments = 'lint --use-stdin'
+        self.lint(filename, file)
```
Comment (Adrianzatreanu) [inline bears/swift/SwiftLintBear.py]: newline here too :)

**Comment by Adrianzatreanu** [comment]
What's wrong with the bear? It seems good. Anyway, could you run ``coala`` over your code? this might fix the circleCi current errors :)

**Comment by SanketDG** [comment]
The swift tests are getting skipped. AFAIK, Swift doesn't have proper support for linux https://swift.org/blog/swift-2-2-released/

Do we want this just for Mac?

**Comment by AbdealiJK** [comment]
Mac is not used for codecov due to some bug in codecov.
So if its not done in linux or windows - codecov's coverage will decrease.

PS: I think you'd prefer linux over windows ;)

**Comment by Adrianzatreanu** [comment]
marking it as blocked as per issue status https://github.com/coala-analyzer/coala-bears/issues/11

**Comment by sils1297** [comment]
Ahm hey, will this be unblocked at some point in time or should we rather close the PR and issue as wontfix?

**Comment by Adrianzatreanu** [comment]
imo i vouch for closing, this looks like a dead-end to me. not many people will profit of it, and it's not worth it.

**Comment by AbdealiJK** [comment]
But why close it?
Let it be kept open for any swift gurus who want to help contribute

-----Original Message-----

**Comment by sils1297** [comment]
Closing this as we'll probably go the tailor way.

**Pull request closed without merging**